http/tus_handlers.go

```diff
@@ -147,7 +147,8 @@ func tusPostHandler() handleFunc {
 		// Enables the user to utilize the PATCH endpoint for uploading file data
 		registerUpload(file.RealPath(), uploadLength)
 
-		w.Header().Set("Location", "/api/tus/"+r.URL.Path)
```

Comment:
I think we should add this back. The problem from the double slash is that r.URL.Path contains the slash, or not? That creates the double slash issue.

Reply:
@ramiresviana maybe `"/api/tus" + r.URL.Path` would do the trick